adafruit_azureiot/device_registration.py

```diff
@@ -13,19 +13,16 @@
 * Author(s): Jim Bennett, Elena Horton
 """
 
-import gc
 import json
 import time
-import adafruit_requests as requests
 import adafruit_logging as logging
 from adafruit_logging import Logger
+import adafruit_minimqtt.adafruit_minimqtt as minimqtt
+from adafruit_minimqtt.adafruit_minimqtt import MQTT
```

Comment:
Why are there two imports for minimqtt? Could you use the `minimqtt` import from L220 and reference the `minimqtt.MQTT` class from there?

Reply:
This was copy/pasted from the iot_mqtt.py file. Fixed here and there as well.

Reply:
Thanks